pkg/adapter/nimbus-netpol/manager/netpols_manager.go

```diff
@@ -34,31 +33,28 @@ var (
 )
 
 func init() {
-	utilruntime.Must(intentv1.AddToScheme(scheme))
+	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	utilruntime.Must(netv1.AddToScheme(scheme))
 	k8sClient = k8s.NewOrDie(scheme)
 }
 
 func Run(ctx context.Context) {
+
+	// Watch NimbusPolicies only, and not ClusterNimbusPolicies as NetworkPolicy is
+	// namespaced scoped
 	npCh := make(chan common.Request)
 	deletedNpCh := make(chan common.Request)
-	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh)
-
-	clusterNpChan := make(chan string)
-	deletedClusterNpChan := make(chan string)
-	go globalwatcher.WatchClusterNimbusPolicies(ctx, clusterNpChan, deletedClusterNpChan)
+	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh, "SecurityIntentBinding", "ClusterSecurityIntentBinding")
```

Comment:
Why does this need to watch for both SIB and CSIB CRs?

Reply:
Please note that the function is watching Nimbus Policies, and not SIB/CSIB CRs